app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -29,18 +29,16 @@ def failure(message, extra = nil)
       FormResponse.new(**form_response_params)
     end
 
-    # @param [DocAuth::Response,
-    #   DocumentCaptureSessionResult] response
-    def extract_pii_from_doc(user, response, store_in_session: false)
+    def extract_pii_from_doc(user, store_in_session: false)
```

Comment:
I was confused about referring to a `DocumentCaptureSessionResult` object as a `response`, thanks for changing that.

Reply:
This will remove the misleading info. 👍